src/main/java/org/spongepowered/common/data/builder/manipulator/mutable/entity/ScreamingDataBuilder.java

```diff
@@ -55,8 +52,8 @@ public ScreamingData create() {
 
     @Override
     public Optional<ScreamingData> createFrom(DataHolder dataHolder) {
-        if (dataHolder.supports(Keys.IS_SCREAMING)) {
-            return Optional.<ScreamingData>of(new SpongeScreamingData(dataHolder.get(Keys.IS_SCREAMING).get()));
+        if (dataHolder instanceof EntityEnderman) {
```

Comment:
I don't think it should be restricted to just enderman. While they are the only ones that have this at the moment it'd be nice to be able to use this a mod-added entity that may have this tag.